docs/zh/latest/FAQ.md

```diff
@@ -501,3 +501,13 @@ $ acme.sh --renew --domain demo.domain
 ```
 
 详细步骤，可以参考博客 https://juejin.cn/post/6965778290619449351
+
+## 如何在kylin v10 国产操作系统运行apisix
```

Comment:
Is this problem confirmed by the official kylin v10 developer?
We don't want to be in trouble for spreading incorrect message about their OS.

Reply:
Officials don't know about this problem, it's just my adaptation to the kylin v10 OS.

Reply:
Maybe we can submit a PR that supporting specifying the `user`, but don't mention a specific OS name as it's not confirmed by the official.

Reply:
Need I submit a new PR  or  change FAQ file in this PR ?

Reply:
I think we can remove the FAQ.

Reply:
But in the Chinese arm OS , if we do not change this config, the apisix cannot run . It will throw "require faild" error like issue #4157.  So should we write something in the FAQ to tell someone who uses apisix in Chinese arm OS how to resolve the problem.

Reply:
We need an official confirm for the FAQ.

Reply:
I have removed this FAQ and committed it.